app/models/pull.rb

```diff
@@ -734,6 +734,18 @@ def mail_subject
     "[#{project.name}] #{subject} (##{id})"
   end
 
+  def merge_commit_message
+    "Merge pull request \"##{id}\":/pulls/#{id} from #{commit_head}"
```

Comment:
It gives a message like this:

```
Merge pull request "#35":/pulls/35 from use-case/not-merged
```

Maybe use the following message:

```suggestion
    "Merge pull request ##{id} from #{commit_head}\n\n#{subject}"
```